pymumble_py3/soundoutput.py

```diff
@@ -100,6 +100,8 @@ def send_audio(self):
             sequence = VarInt(self.sequence).encode()
 
             udppacket = struct.pack('!B', header | self.target) + sequence + payload
+            if self.mumble_object.positional:
+                udppacket += struct.pack("fff", self.mumble_object.positional[0], self.mumble_object.positional[1], self.mumble_object.positional[2])
```

Comment:
Does the positional need to be send on every sound send to the server ?

Reply:
Actually I am not sure how the server handles it if some sounds miss the positional data. I think it will assume the positional data is non-existent on it and will play it without spatialization, so I would say yes, it needs to be set all the time but I did not test it.

Reply:
AFAIK yes. You need to transmit them with all the packets. Mumble will assume the positional vector to be (0,0,0) if one packet doesn't come with a positional message. This is by no means economy, but that's how mumble works. And perhaps this should be fixed in the future.